contracts/index.js

```diff
@@ -45,43 +45,37 @@ exports.LIB_ARTIFACT_UTILS_ADDRESS =
 /**
  * The name of the network where these contracts are deployed.
  */
-exports.NETWORK = 'gnosis_optimism';
+exports.NETWORK = 'hardhat';
```

Comment:
we don't want any changes to `index.d.ts.map`, `index.js`, `index.js.map` because that overwrites our gnosis addresses